src/styles/blocks/page.scss

```diff
@@ -0,0 +1,34 @@
+.page {
+  font-family: "Poppins", sans-serif;
+  color: #fff;
```

Comment:
use variables for all colors

Reply:
add additional property for `font-family` `"Poppins", Arial, Helvetica, sans-serif;`